### PR TITLE
Added mailing_list page.

### PR DESCRIPTION
**Pull request by BobbyRBruce**
First of all: This page is likely to change as we migrate our mail services, though this should serve as a decent template to change going forward.

Secondly: Do we want to keep Quora as communication channel?

### CODE REVIEW

File: _pages/mailing_list.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: toc
+title: Mailing Lists
+permalink: /mailing-lists/
```
Comment (powerjg) [inline _pages/mailing_list.md]: `-` or `_`? What's most common right now?

File: _pages/mailing_list.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[Stack Overflow](https://stackoverflow.com) can be used to crowd-source
+solutions to gem5 related problems. The Stack Overflow `gem5` tag should be
+used: <https://stackoverflow.com/questions/tagged/gem5>). Please make sure
+that your question complies with the [site guidelines](
+https://stackoverflow.com/help/asking) before posting. Interested users can opt
+to receive email notifications for such questions as explained at
+<https://meta.stackexchange.com/questions/25224/email-notifications-for-new-questions-matching-specific-tags>).
+
+Quora is occasionally used : <https://www.quora.com/topic/Gem5-Simulator>.
```
Comment (powerjg) [inline _pages/mailing_list.md]: lololol. We should remove this, IMO

Comment (BobbyRBruce) [reply _pages/mailing_list.md]: Agreed

Comment (BobbyRBruce) [reply _pages/mailing_list.md]: Removed

File: _pages/mailing_list.md (hunk 1 above)
Comment (BobbyRBruce) [reply _pages/mailing_list.md]: Changed to `_` for consistency.

**Pull request merged**